src/rpc/server/linux_io_uring.zig

```diff
@@ -10,6 +10,8 @@ const IoUring = std.os.linux.IoUring;
 
 const LOGGER_SCOPE = "rpc.server.linux_io_uring";
 
+const HTTP_VERSION: std.http.Version = .@"HTTP/1.0";
+
```

Comment:
Do we definitely want 1.0? If yes, a comment would be good. I think 1.1 is more commonly used

Reply:
We don't really support or use any HTTP/1.1 features as of now, and I don't really think the server should advertise itself as being 1.0 if it doesn't make any effort to be.